rhc.spec.in

```diff
@@ -69,11 +69,29 @@ make PREFIX=%{_prefix} \
      'PROVIDER=@PROVIDER@' \
      install
 
+%post
+%systemd_post rhc-canonical-facts.timer
+if [ $1 -eq 1 ]; then
+     systemctl daemon-reload
+     systemctl start rhc-canonical-facts.timer
+fi
+
+%preun
+%systemd_preun rhc-canonical-facts.timer
+
+%postun
+%systemd_postun_with_restart rhc-canonical-facts.timer
+if [ $1 -eq 0 ]; then
+     systemctl daemon-reload
+fi
```

Comment:
I can confirm that this works, but dnf still prints warning, when rhc RPM is removed. We probably cannot do anything with that.